events/event.html

```diff
@@ -0,0 +1,128 @@
+                <path
+                  d="M22.3465 7.79492H1.65354C1.215 7.79492 0.794411 7.96913 0.484311 8.27923C0.174212 8.58933 0 9.00992 0 9.44847C0 9.88701 0.174212 10.3076 0.484311 10.6177C0.794411 10.9278 1.215 11.102 1.65354 11.102H22.3465C22.785 11.102 23.2056 10.9278 23.5157 10.6177C23.8258 10.3076 24 9.88701 24 9.44847C24 9.00992 23.8258 8.58933 23.5157 8.27923C23.2056 7.96913 22.785 7.79492 22.3465 7.79492Z" />
+                <path
+                  d="M24 17.2444C24 16.8058 23.8258 16.3852 23.5157 16.0751C23.2056 15.765 22.785 15.5908 22.3465 15.5908H1.65354C1.215 15.5908 0.794411 15.765 0.484311 16.0751C0.174212 16.3852 0 16.8058 0 17.2444C0 17.6829 0.174212 18.1035 0.484311 18.4136C0.794411 18.7237 1.215 18.8979 1.65354 18.8979H22.3465C22.785 18.8979 23.2056 18.7237 23.5157 18.4136C23.8258 18.1035 24 17.6829 24 17.2444Z" />
+              </svg>
+              <!-- exit menu icon -->
+              <svg
+                width="24"
+                height="24"
+                viewBox="0 0 24 24"
+                xmlns="http://www.w3.org/2000/svg"
+                class="menu-exit-icon hidden">
+                <path
+                  d="M1.7416 23.9993C1.29575 23.9993 0.849903 23.8293 0.509942 23.4894C-0.169981 22.8094 -0.169981 21.7067 0.509942 21.026L21.0267 0.509942C21.7067 -0.169981 22.8094 -0.169981 23.4901 0.509942C24.17 1.18986 24.17 2.29265 23.4901 2.97327L2.97257 23.4894C2.63261 23.8293 2.18745 23.9993 1.7416 23.9993Z" />
+                <path
+                  d="M22.2577 24C21.8118 24 21.366 23.83 21.026 23.4901L0.509942 2.97327C-0.169981 2.29334 -0.169981 1.19056 0.509942 0.509942C1.18986 -0.169981 2.29265 -0.169981 2.97327 0.509942L23.4901 21.0267C24.17 21.7067 24.17 22.8094 23.4901 23.4901C23.1494 23.83 22.7035 24 22.2577 24Z" />
+              </svg>
+            </button>
+
+            <!-- Page Links -->
+            <ul id="nav-links" class="nav-links">
+              <li><a href="./about.html">About US</a></li>
+              <li><a href="#">Our Impact</a></li>
+              <li><a href="./values.html">Our Values</a></li>
+              <li><a href="./events.html">Events</a></li>
+              <li><a href="./our-team.html">Our Team</a></li>
+              <li><a href="./contact.html">Contact</a></li>
+              <!-- Mobile Support Button -->
+              <li class="mobile-support-btn"><a href="#">Support US</a></li>
+            </ul>
+          </div>
+
+          <!-- desktop support btn -->
+          <a href="#" class="btn-main desktop-support-btn">Support US</a>
+        </div>
+      </div>
+    </nav>
+    <!-- Hero -->
+    <section class="event-detail-container">
```

Comment:
Not sure if this section should be in the hero section. it may make more sense to have a `<main>` element containing all the event sections outside of the header to keep things consistent.
I think structuring pages w `<header>`, `<main>`, and `<footer>`  will keep things consistent between our code.

This also may need to be adjusted in  events and programs page